core/instance/Instance.js

```diff
@@ -655,10 +664,19 @@ class Instance extends EventEmitter {
             var bitBuffer = createSnapshotBuffer(snapshot, this.config)
             var buffer = bitBuffer.toBuffer()
 
-            if (client.connection.readyState === 1) {
+            // There should probably be a check for backPressure here
```

Comment:
if we ever run into this issue, we'd have to choose to either buffer our messages temporarily or drop them. i have no idea what happens if nengi drops a frame, and i guess we could find out, but that's probably not a today problem.